tools/licenses/lib/main.dart

```diff
@@ -357,37 +357,68 @@ class RepositoryBlankLicenseFile extends RepositorySingleLicenseFile {
   License licenseOfType(LicenseType type) => null;
 }
 
-class RepositoryOkHttpLicenseFile extends RepositorySingleLicenseFile {
-  RepositoryOkHttpLicenseFile(RepositoryDirectory parent, fs.TextFile io)
+class RepositoryCatapultApiClientLicenseFile extends RepositorySingleLicenseFile {
+  RepositoryCatapultApiClientLicenseFile(RepositoryDirectory parent, fs.TextFile io)
     : super(parent, io, _parseLicense(io));
 
   static final RegExp _pattern = new RegExp(
-    r'^((?:.|\n)*)\n'
-    r'Licensed under the Apache License, Version 2\.0 \(the "License"\);\n'
-    r'you may not use this file except in compliance with the License\.\n'
-    r'You may obtain a copy of the License at\n'
-    r'\n'
-    r'   (http://www\.apache\.org/licenses/LICENSE-2\.0)\n'
-    r'\n'
-    r'Unless required by applicable law or agreed to in writing, software\n'
-    r'distributed under the License is distributed on an "AS IS" BASIS,\n'
-    r'WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.\n'
-    r'See the License for the specific language governing permissions and\n'
-    r'limitations under the License\.\n*$',
-    caseSensitive: false
+    r' *Licensed under the Apache License, Version 2\.0 \(the "License"\);\n'
+    r' *you may not use this file except in compliance with the License\.\n'
+    r' *You may obtain a copy of the License at\n'
+    r' *\n'
+    r' *(http://www\.apache\.org/licenses/LICENSE-2\.0)\n'
+    r' *\n'
+    r' *Unless required by applicable law or agreed to in writing, software\n'
+    r' *distributed under the License is distributed on an "AS IS" BASIS,\n'
+    r' *WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.\n'
+    r' *See the License for the specific language governing permissions and\n'
+    r' *limitations under the License\.\n',
+    multiLine: true,
```

Comment:
why doesn't the usual automatic apache license detection work here?

Reply:
It's the same situation as okhttp used to have.  This file doesn't contain the full apache license.  Just this shorter text and a link to the full license.